Tests/QuickTests/QuickTests/FunctionalTests/ItTests.swift

```diff
@@ -101,6 +101,26 @@ class FunctionalTests_ItSpec: QuickSpec {
 
                 it("should throw an exception with the correct message after this 'it' block executes") {  }
             }
+
+            describe("implicit error handling") {
+                enum ExampleError: Error {
+                    case error
+                }
+
+                func nonThrowingFunc() throws { }
+
+                func throwingFunc() throws {
+                    throw ExampleError.error
+                }
+
+                it("supports calling functions marked as throws") {
+                    try nonThrowingFunc()
+                }
+
+                it("supports calling functions that actually throws") {
```

Comment:
Would something like how [focused tests](https://github.com/Quick/Quick/blob/master/Tests/QuickTests/QuickFocusedTests/FocusedTests.swift) are executed work? I'm not as up to speed on the Quick side of the code for bootstrapping unfortunately, perhaps other @Quick/core can answer this better.